src/app/service/scoap-squad/ScoapNotion.ts

```diff
@@ -0,0 +1,211 @@
+import { Client as NotionClient } from '@notionhq/client';
+import {
+	InputPropertyValueMap,
+	PropertyMap,
+} from '@notionhq/client/build/src/api-endpoints';
+import {
+	MultiSelectPropertyValue,
+	SelectPropertyValue,
+	DatePropertyValue,
+	TitlePropertyValue,
+	RichTextPropertyValue,
+	ParagraphBlock,
+} from '@notionhq/client/build/src/api-types';
+import constants from '../constants/constants';
+
+const notion = new NotionClient({ auth: process.env.NOTION_TOKEN });
+const databaseId = process.env.NOTION_SCOAP_SQUAD_DB_ID;
+
+const notionDatabaseProperties = async () => {
+	const db_properties = await getNotionDatabaseProperties();
+	return db_properties;
+};
+
+export const updateScoapOnNotion = async (page_id: string, inputs: Record<string, any>) => {
+	const m_select_options = retrieveMultiSelectOptions(await notionDatabaseProperties(), constants.SCOAP_SQUAD_NOTION_FIELDS.scoap_squad_discord_handles.field_name, inputs.discord_tags);
```

Comment:
This is probably going to sound (and be) really annoying, but will you change the variable declarations from snake_case to camelCase? With the exception of the variables in the `constants` folder. Let's keep the codebase as consistent as possible.

Reply:
done